bpmn2stamp-converter/src/test/java/cz/cvut/kbss/bpmn2stamp/wip/bpmn2java/ConverterXmlFileReaderTest.java

```diff
@@ -94,6 +94,22 @@ public void mapAndWriteTestNew() {
 //        );
     }
 
+    @Test
+    public void testReadActorMappingFile() {
+        ConverterMappingService converterMappingService = new ConverterMappingService(
+                "prefix-bpmn",
+                "prefix-organization-structure",
+                "prefix-pre-stamp"
+        );
+        ActorMappings actorsFromXml = converterMappingService.readActorMappingFile(new File("src/test/java/cz/cvut/kbss/bpmn2stamp/wip/bpmn2java/data/actor_mapping_1.xml"));
```

Comment:
IMHO, we should rather use resources that path to file so it works even if executed from parent project or elsewhere.  Is it complicated to use like that?

Reply:
data --> should be `/src/test/resources`

Reply:
fixed